internal/pkg/describe/static_site.go

```diff
@@ -67,3 +78,106 @@ func (d *StaticSiteDescriber) URI(envName string) (URI, error) {
 		AccessType: URIAccessTypeInternet,
 	}, nil
 }
+
+// Describe returns info of a static site.
+func (d *StaticSiteDescriber) Describe() (HumanJSONStringer, error) {
+	environments, err := d.store.ListEnvironmentsDeployedTo(d.app, d.svc)
+	if err != nil {
+		return nil, fmt.Errorf("list deployed environments for service %q: %w", d.svc, err)
+	}
+	var routes []*WebServiceRoute
+	for _, env := range environments {
+		uri, err := d.URI(env)
+		if err != nil {
+			return nil, fmt.Errorf("retrieve service URI: %w", err)
+		}
+		if uri.AccessType == URIAccessTypeInternet {
+			routes = append(routes, &WebServiceRoute{
+				Environment: env,
+				URL:         uri.URI,
+			})
+		}
+	}
+	resources := make(map[string][]*stack.Resource)
+	if d.enableResources {
+		for _, env := range environments {
+			svcDescr, err := d.initWkldStackDescriber(env)
+			if err != nil {
+				return nil, err
+			}
+			stackResources, err := svcDescr.StackResources()
+			if err != nil {
+				return nil, fmt.Errorf("retrieve service resources: %w", err)
+			}
+			resources[env] = stackResources
+		}
+	}
+	return &staticSiteDesc{
+		Service:   d.svc,
+		Type:      manifestinfo.StaticSiteType,
+		App:       d.app,
+		Routes:    routes,
+		Resources: resources,
+
+		environments: environments,
+	}, nil
+}
+
+// Manifest returns the contents of the manifest used to deploy a static site stack.
+// If the Manifest metadata doesn't exist in the stack template, then returns ErrManifestNotFoundInTemplate.
+func (d *StaticSiteDescriber) Manifest(env string) ([]byte, error) {
+	cfn, err := d.initWkldStackDescriber(env)
+	if err != nil {
+		return nil, err
+	}
+	return cfn.Manifest()
+}
+
+// staticSiteDesc contains serialized parameters for a static site.
+type staticSiteDesc struct {
+	Service   string               `json:"service"`
+	Type      string               `json:"type"`
+	App       string               `json:"application"`
+	Routes    []*WebServiceRoute   `json:"routes"`
+	Resources deployedSvcResources `json:"resources,omitempty"`
+
+	environments []string `json:"-"`
+}
+
+// JSONString returns the stringified backendService struct with json format.
+func (w *staticSiteDesc) JSONString() (string, error) {
+	b, err := json.Marshal(w)
+	if err != nil {
+		return "", fmt.Errorf("marshal static site description: %w", err)
+	}
+	return fmt.Sprintf("%s\n", b), nil
+}
+
+// HumanString returns the stringified backendService struct with human readable format.
+func (w *staticSiteDesc) HumanString() string {
+	var b bytes.Buffer
```

Comment:
i wonder if https://pkg.go.dev/strings#Builder makes more sense 💭

Reply:
I guess it'll do the same thing but what's extra thing we can get for making this change 🤔

Reply:
more performant if you are only writing strings! (but yeah, not a huge deal, so just a nit)